code/_onclick/drag_drop.dm

```diff
@@ -68,9 +68,6 @@
 
 /obj/item/proc/onMouseUp(object, location, params, mob)
 	return
-/obj/item/gun
```

Comment:
hate when they do this, as cntrl+clicking a `/obj/item/gun/child` trying to find the parent links to here instead of the actual parent. 

Apparentely they managed to trick lamiales in their "nudges you with" pr, because they added the item_flag here, even though there is already an item_flag define on `/obj/item/gun` at its true file.

Reply:
oh yea, this really annoys me as well. Modular code like this needs to be done after all the other definitions